src/routes/community.md

```diff
@@ -1,5 +1,23 @@
 <script lang="ts">
-  import {
+  import {s
+    OutboundLink
+  } from 'carbon-components-svelte'
+
+  import OutlineHelper from '$components/OutlineHelper.svelte'
+
+</script>
+
+<OutlineHelper />
+
+<div class="markdown-generated">
+
+# Community
+
+* <OutboundLink  href="https://fission.codes/discord">Join the Fission Discord server</OutboundLink>
+* <OutboundLink  href="https://talk.fission.codes/tag/ucan">Fission Community Forum</OutboundLink>
+
+</div>
+=======
```

Comment:
yeah, looking at this now I assume some of this is pilot error but I should have opted for resolving these conflicts locally instead of using Github's tool.